### DISCUSSION TITLE
Sync snapcast clients with server MPD USB DAC?

### DISCUSSION THREAD

**Discussion by wkuhns** [snapcast/snapcast#1309] [Q&A] [answered]
I have a multiroom setup using mpd on the music server, configured with two audio outputs: a USB DAC for very high fidelity in the main living room, and snapserver to feed clients in other rooms. Everything works, but there's a significant sync issue between the living room and the remotes. Is there a way to sync the mpd USB output with the snapclients? I assume that the delay is caused by necessary buffering for snapcast.
![stereo](https://github.com/user-attachments/assets/b74d54e3-24c8-4944-999f-2ab6997123f1)

**Comment by vjdw** [comment, marked as answer]
This is quite similar to my setup, and I had the same problem. You can fix it by running Snapcast Client also on the music server that has the DAC plugged into it, and disable the audio output from MPD to the DAC. Then all audio is going through snapcast and so it's all in sync.

**Reply by wkuhns** [reply]
Thanks - I wondered if that was the answer. It *seems* to work perfectly now.
